static/js/utils/app/explore_utils.ts

```diff
@@ -61,3 +63,63 @@ export function getTopics(
   }
   return topicList;
 }
+
+/**
+ * Returns true if highest point coverage for a given parent entity, child type,
+ * and variable is equal to the latest observation dates for the same parent
+ * entity, child type, variable combination.
+ *
+ * Put another way, returns true if calling /api/observations/point/within with
+ * DATE_HIGHEST_COVERAGE gives the same result as DATE_LATEST
+ *
+ * @returns boolean
+ */
+export async function highestCoverageDatesEqualLatestDates(
+  parentEntity: string,
+  childType: string,
+  variables: string[]
+): Promise<boolean> {
+  const highestCoverageObservations =
+    await datacommonsWebClient.getObservationsPointWithin({
+      parentEntity,
+      childType,
+      variables,
+      date: DATE_HIGHEST_COVERAGE,
+    });
+  const latestObservations =
+    await datacommonsWebClient.getObservationsPointWithin({
+      parentEntity,
+      childType,
+      variables,
+      date: DATE_LATEST,
+    });
+
+  // Return false if we find any "latest observation dates" that differ from the
+  // "highest coverage date"
+  const highestCoverageVariableDcids = Object.keys(
+    highestCoverageObservations.data
+  );
+  for (let variableDcid of highestCoverageVariableDcids) {
+    const entityDcid = Object.keys(
+      highestCoverageObservations.data[variableDcid]
+    ).pop();
+    if (!entityDcid) {
+      continue;
+    }
+    const highestCoverageDate =
+      highestCoverageObservations.data[variableDcid][entityDcid].date;
+    const latestObservationEntityDcids = Object.keys(
+      latestObservations.data[variableDcid]
```

Comment:
should check that there exists variableDcid in latestObservations.data first or `latestObservations.data[variableDcid] || {}`

Reply:
done